website/docs/cli/configuration/markdown-styling.mdx

```diff
@@ -0,0 +1,204 @@
+| `indent` | number | Indentation level |
+
+### Element-Specific Properties
+
+#### Document
+- Base styling for all text content
+- Supports all common properties
+
+#### Headings (H1-H6)
+- Individual styling for each heading level
+- H1 supports additional `background_color` property
+- All heading levels support `margin` for vertical spacing
+
+#### Code Blocks
+- Styling for multi-line code blocks
+- Supports `margin` for visual separation
+- Color applies to the entire block
+
+#### Block Quotes
+- Styling for quoted text
+- `indent` property controls quote indentation
+- Supports all common properties
+
+#### Links
+- Styling for hyperlinks
+- `underline` property specifically for links
+- Color applies to both link text and underline
+
+## Default Styles
+
+If no custom styles are configured, Atmos uses a built-in default theme related to the default atmos brand colors:
+
+```yaml
+# Built-in default theme
+document:
+  color: "#FFFFFF"  # White text
+heading:
+  color: "#00A3E0"  # Blue headings
+  bold: true
+h1:
+  color: "#FFFFFF"
+  background_color: "#9B51E0"  # Purple background
+  bold: true
+code_block:
+  color: "#00A3E0"  # Blue code
+link:
+  color: "#00A3E0"
+  underline: true
+```
+
+## Terminal Compatibility
+
+The markdown styling feature requires a terminal that supports:
+- 24-bit true color (for hex color codes)
+- ANSI escape sequences
+- Unicode characters
+
+Most modern terminals support these features. For terminals with limited color support, Atmos will automatically adjust colors to the closest available match.
+
+## Examples
+
+### Error Messages
+Custom styling makes error messages more readable:
+
+```yaml
+settings:
+  markdown:
+    h1:
+      color: "#FFFFFF"
+      background_color: "#FF0000"  # Red background for errors
+      bold: true
+    code_block:
+      color: "#00FFFF"  # Cyan for code examples
+```
+
+### Help Text
+Enhance help text readability:
+
+```yaml
+settings:
+  markdown:
+    heading:
+      color: "#FFD700"  # Golden headings
+      bold: true
+    code_block:
+      color: "#00FFFF"  # Cyan code examples
+    link:
+      color: "#FFD700"
+      underline: true
+```
+
+## Best Practices
+
+1. **Color Contrast**: Ensure sufficient contrast between text and background colors for readability.
+2. **Consistent Styling**: Use a consistent color scheme across different elements.
+3. **Terminal Support**: Test your styling in different terminals to ensure compatibility.
+4. **Accessibility**: Consider color-blind users when choosing your color scheme.
+
+## Troubleshooting
+
+If styles aren't applying correctly:
+
+1. Verify your terminal supports true color:
+```bash
+$ echo $COLORTERM
+```
+Should output `truecolor` or `24bit`.
+
+2. Check your `atmos.yaml` is properly formatted:
+- Colors must be in hex format
+- Boolean values must be `true` or `false`
+- Numbers must be integers
+
+3. Try removing custom styles to see if default styles work.
```

Comment:
~~~suggestion
1. **Verify Terminal Supports True Color:**

    - **Check `$COLORTERM`:**
      ```bash
      echo $COLORTERM
      ```
      **Expected Output:** `truecolor` or `24bit`

    - **Check `$TERM`:**
      ```bash
      echo $TERM
      ```
      **Recommended Values:** `xterm-256color`, `xterm-direct`, `xterm-truecolor`

2. **Ensure Your Terminal Emulator Supports True Color:**

    - Use a terminal emulator known for true color support (e.g., Terminal.app, iTerm2, Windows Terminal, etc).

3. **Configure Environment Variables Correctly:**

    - Set `$TERM` to a value that supports true color:
      ```bash
      export TERM=xterm-256color
      ```
      Add this to your shell’s configuration file (`~/.bashrc`, `~/.zshrc`, etc.) to make it permanent.

4. **Validate `atmos.yaml` Configuration:**

    - Ensure colors are in hex format, boolean values are `true`/`false` (not quoted strings), and numbers are integers.
    - Use a YAML linter to validate the syntax.
    - Try removing custom styles to see if default styles work.


~~~